tests/test_alembic_dialect.py

```diff
@@ -1,3 +1,6 @@
+import pytest
+
+pytest.importorskip('alembic')
```

Comment:
i don't think alembic should be required to run the tests as it isn't required to use the dialect